### DISCUSSION TITLE
Following Lesson 1 on keystonejs.com results in error at "Run keystone" step:  Error: Cannot use GraphQLInputObjectType

### DISCUSSION THREAD

**Discussion by glennflanagan** [keystonejs/keystone#7920] [Questions] [answered]
I was following the [Lesson 1 Steps](https://keystonejs.com/docs/walkthroughs/lesson-1) and got as far as the [Run Keystone step](https://keystonejs.com/docs/walkthroughs/lesson-1#run-keystone) and encountered this error:

```
Error: Cannot use GraphQLInputObjectType "{ name: "IDFilter", description: undefined, extensions: {}, astNode: undefined, extensionASTNodes: [], _fields: [function bound defineInputFieldMap] }" from another module or realm.

Ensure that there is only one instance of "graphql" in the node_modules
directory. If different versions of "graphql" are the dependencies of other
relied on modules, use "resolutions" to ensure only one version is installed.

https://yarnpkg.com/en/docs/selective-version-resolutions

Duplicate "graphql" modules cannot be used at the same time since different
versions may have different capabilities and behavior. The data from one
version used in the function from another could produce confusing and
```

## Reproduction steps
Clone project https://github.com/glennflanagan/keystone-6-lesson-1 
Run `npm i`
Run `npm run dev`
(Or view https://codesandbox.io/p/github/glennflanagan/keystone-6-lesson-1/csb-wmezk5/draft/gallant-golick)

## Expected Behaviour
Lesson states
> In a few seconds your terminal will provide you with you a link to the Keystone Admin UI at http://localhost:3000/

### Screenshot
<img width="1040" alt="Screenshot 2022-09-12 at 22 27 33" src="https://user-images.githubusercontent.com/5485364/189761775-4404c991-78b3-4c8f-985e-148b7fe26878.png">

Node version: `v16.17.0`
package.json:
```
{
  "name": "keystone-lesson-1",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1",
    "dev": "keystone dev"
  },
  "author": "",
  "license": "ISC",
  "dependencies": {
    "@keystone-6/core": "^2.2.0",
    "typescript": "^4.8.3"
  }
}
```

**Comment by dcousens** [comment, marked as answer]
Hi @glennflanagan,

Thanks for trying out Keystone 6!
This is a recent problem that has been discussed and different workarounds posted in https://github.com/keystonejs/keystone/issues/7816.

The problem will be resolved in the next release as we upgrade to `graphql@16`.
